DigitalLearningSolutions.Data.Tests/Services/CentresServiceTests.cs

```diff
@@ -87,5 +87,21 @@ public void GetAllCentreSummariesForSuperAdmin_calls_dataService_and_returns_all
                 .Should()
                 .HaveCount(10);
         }
+
+        [Test]
+        public void GetAllCentreSummariesForMap_calls_dataService_and_returns_all_summary_details()
+        {
+            // Given
+            var centres = Builder<CentreSummaryForMap>.CreateListOfSize(10).Build();
+            A.CallTo(() => centresDataService.GetAllCentreSummariesForMap()).Returns(centres);
+
+            // When
+            var result = centresService.GetAllCentreSummariesForMap();
+
+            // Then
+            result
+                .Should()
+                .HaveCount(10);

```

Comment:
not important but this looks weirdly auto-formatted?

Reply:
Also do we want to check the result is equivalent to the centres?